Python/backtracking/N-queens.py

```diff
@@ -0,0 +1,55 @@
+# You are given N, and for a given N x N chessboard, find a way to place N queens such that no queen can attack any other queen on the chess board. A queen can be killed when it lies in the same row, or same column, or the same diagonal of any of the other queens. You have to print all such configurations.
```

Comment:
Gosh! That's a very very long comment! 

We usually assume that the Code should be formatted enough to be readable enough. You can try breaking down this Comment into multiple lines.